docs/tutorials/Soil/freezing_front.jl

```diff
@@ -200,14 +200,18 @@ end
 
 init_soil!(Y, coords.z, soil.parameters);
 
-# Create the tendency function, and choose a timestep, integration timespan, etc:
-exp_tendency! = make_exp_tendency(soil)
+# We choose the initial and final simulation times:
```

Comment:
I like the tweaks that add clarity to the code throughout this PR =)

Reply:
thanks for catching the typos!